presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -929,6 +930,19 @@ public HiveClientConfig setFailOnCorruptedParquetStatistics(boolean failOnCorrup
         return this;
     }
 
+    public boolean isIgnoreBinaryStatisticsForVarchars()
+    {
+        return isIgnoreBinaryStatisticsForVarchars;
+    }
+
+    @Config("hive.parquet.ignore-binary-statistics-for-varchars")
+    @ConfigDescription("Ignore the incorrect binary-statistics for varchars when pushdown predicates")
```

Comment:
I'd change this to 

`Ignore binary statistics for varchar columns when scanning Parquet files`